translations/de.po

```diff
@@ -1810,7 +1810,7 @@ msgstr "Suche auf die aktuelle Seite und Unterseite(n) einschränken"
 #. menu item
 #: zim/gui/searchdialog.py:150 zim/plugins/scoreeditor.py:68
```

Comment:
Took me a while to notice, but the translation is correct - for the search result scoring. The plugin needs to be modified.

Reply:
I suggest changing the visible string to "Music Score", because it's an item in a menu and should be unambiguous in any language

Reply:
@dehesselle yes it does:

https://github.com/zim-desktop-wiki/zim-desktop-wiki/blob/a9917c94a033a2ebf57bcd4ee111f023b14b03a0/translations/de.po#L1809-L1813